### ISSUE TITLE
ReferenceError: Class is not defined

### ISSUE THREAD

**Issue by nucleartide** [marionettejs/backbone-metal#28]
Hey @thejameskyle, big fan of your Backbone-related projects. I'm currently using something very similar to marionette-wires at work, and it's working great!

So I don't have time to dig into this at work, but a fresh install of backbone-metal will throw this error:

In node:

```
/home/jason/Stuff/Dump/backbone/node_modules/backbone-metal/dist/backbone-metal.js:496
    Class: Class,
           ^
ReferenceError: Class is not defined
    at /home/jason/Stuff/Dump/backbone/node_modules/backbone-metal/dist/backbone-metal.js:496:12
    at /home/jason/Stuff/Dump/backbone/node_modules/backbone-metal/dist/backbone-metal.js:2:83
    at Object.<anonymous> (/home/jason/Stuff/Dump/backbone/node_modules/backbone-metal/dist/backbone-metal.js:3:3)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/jason/Stuff/Dump/backbone/main.js:1:63)
```

And in browser, compiled with browserify:

![screenshot from 2015-07-24 14 05 42](https://cloud.githubusercontent.com/assets/914228/8881354/22ec113c-320d-11e5-9ca6-bcc8a42db046.png)

**Comment by jamiebuilds** [comment]
Looks like an issue with esperanto:
- https://github.com/marionettejs/backbone-metal/blob/master/dist/backbone-metal.js#L117
- https://github.com/marionettejs/backbone-metal/blob/master/dist/backbone-metal.js#L496

cc @paulfalgout

**Comment by jamiebuilds** [comment]
Fixed and published in v0.6.2
